notifications.bs

```diff
@@ -108,19 +108,42 @@ the <a>notification</a>, but then it should have less visual priority than the
 is not otherwise accessible to the end user, especially since notification platforms that do not
 support these features might ignore them.
 
-<p>A <a>notification</a> has an associated list of
-zero or more <dfn for=notification lt=action id=actions>actions</dfn>. Each
-<a for=notification>action</a> has an associated
-<dfn for=action id=action-title>title</dfn> and <dfn for=action id=action-name>name</dfn> and
-<em>can</em> have an associated <dfn for=action>icon URL</dfn> and
-<dfn for=action>icon resource</dfn>. Users may activate actions, as alternatives to
-activating the notification itself. The user agent must determine the
-<dfn>maximum number of actions</dfn> supported, within the constraints of the
-notification platform.
+<p>A <a>notification</a> has an action list (a <a for=/>list</a> of
+<dfn for=notification lt=action id=actions>actions</dfn>). It is initially empty.
```

Comment:
This should become a reference `<a for=notification>action</a>` and then this `<dfn>` can replace the `<dfn>` below so we just have "action" and not "notification action".